src/fragments/gen2/quickstart/deploy-and-host.mdx

```diff
@@ -8,8 +8,8 @@ If you already have your project remotely hosted in a Git repository, you can sk
 
 ### Connect branch in the Amplify console
 
-1. To get started with Gen 2, log in to the [AWS console](https://console.aws.amazon.com/console/home?#amplify) and navigate to your preferred Region. (The Amplify console is available in [19 Regions](https://docs.aws.amazon.com/general/latest/gr/amplify.html#amplify_region)).
-2. If you have never created apps with Amplify before, choose _Create an app_ from the purple banner (top screenshot). If you have Amplify apps, you can find the option under *New app > Build an app (Gen 2)*.
+1. To get started with Gen 2, log in to the [AWS console](https://console.aws.amazon.com/console/home?#amplify) and navigate to your preferred AWS Region. (The Amplify console is available in [19 AWS Regions](https://docs.aws.amazon.com/general/latest/gr/amplify.html#amplify_region)).
+2. From the _Public Preview_ banner, choose _Try Amplify Gen 2_.
 
 ![gen2](/images/gen2/getting-started/console1.png)
```

Comment:
I know this wasn't part of the PR, but could we get a better alt text for this? "gen2" is not really useful for people using screen readers

Reply:
I have updated the alt text, let me know if it better explains the image asset.